cfme/intelligence/reports/ui_elements.py

```diff
@@ -704,10 +697,10 @@ def _click_function():
         return _click_function
 
     def commit(self):
-        self._click_button("Commit expression element changes")
+        self._click_button("Commit")
```

Comment:
@psav yes, because in all streams button's locator is different but all of them contains "Commit" and "Discard".